Nduhiu17/customer-service-dashboard
# #11213232434  Create Navbar features

#### What does this PR do?

- Have Navbar features for the design part

#### Description of Task to be completed?

- Have the following features working.
1. Button to add a new customer
2. Dropdown menu to select the kind of customer

#### How should this be manually tested?

- Clone the repo

-Install Dependencies by npm install

- Run the server by npm start

- Check the Navbar features being displaying on the browser


#### What are the relevant pivotal tracker stories?

- 11213232434 

#### Screenshots (if appropriate)

![screenshot from 2019-01-19 14-11-36](https://user-images.githubusercontent.com/30591881/51426087-8348c580-1bf6-11e9-83f4-fc746efd1615.png)


#11213232434  Create Navbar features
